xml/cap_depl_install_minimal.xml

```diff
@@ -79,8 +81,9 @@
      <title>You must be familiar with &suse; &caasp;</title>
      <para>
         Setting up &suse; &caasp; correctly, and knowledge of basic administration is 
-        essential to a successful &productname; deployment. See
-     the <link xlink:href="https://www.suse.com/documentation/suse-caasp-2/book_caasp_deployment/index.html">&suse; &caasp; 2 Deployment Guide</link>
+        essential to a successful &productname; deployment. See the <link xlink:href=
+        "https://www.suse.com/documentation/suse-caasp-3/book_caasp_deployment/index.html"
```

Comment:
This link is leading to a 404 for me and the link text is still referring to CaaSP 2.

Reply:
Hi @btat, thanks for capturing this. The URL should be fixed now and the text refer to CaaSP v3.